outputs.tf

```
@@ -1,6 +1,6 @@
 output "vpn_gateway_id" {
   description = "ID of the VPN Gateway"
-  value       = join("", aws_vpn_gateway.this.*.id)
+  value       = aws_vpn_gateway.this.*.id
```

the `.*` is part of the join syntax, and an artifact of previously using count on aws_vpn_gateway.
```suggestion
  value       = aws_vpn_gateway.this.id
```